chapters/annotations.tex

```diff
@@ -165,7 +165,9 @@ \subsubsection{Plot Curves}\doublelabel{plot-curves}
 \end{lstlisting}
 
 The mandatory \lstinline!x! and \lstinline!y! expressions are restricted to be
-component references refering to a scalar variable or \lstinline!time!.
+component references refering to a scalar variable or \lstinline!time!. If
+\lstinline!x! or \lstinline!y! does not designate a variable the behavior is
+unspecified.
```

Comment:
I couldn't figure out how to edit the suggestion to include one more row so here is a new one.
```suggestion
component references refering to a scalar variable or \lstinline!time!. It
is an error if \lstinline!x! or \lstinline!y! does not designate a scalar variable.
```
Or should we explicitly mention time again?
```suggestion
component references refering to a scalar variable or \lstinline!time!. It
is an error if \lstinline!x! or \lstinline!y! does not designate a scalar variable or \lstinline!time!.
```

Reply:
To me it seems sufficient to consider `time` a scalar variable in this context, that is, that it doesn't need to be mentioned explicitly.